docs/backstop.md

```diff
@@ -0,0 +1,65 @@
+# BackstopJS Getting Started Guide
+
+## Overview
+
+BackstopJS is currently optional in this project and is used for visual regression testing to ensure consistency in the UI. To get started with BackstopJS, follow the steps below. The official guide can be found at the [BackstopJS GitHub Repository](https://github.com/garris/BackstopJS).
+
+## Goal
+
+Eventually BackstopJS will be implemented in the pipeline process to help catch errors before they make their way to higher environments.
+
+## Prerequisites
+
+Before running BackstopJS, ensure the following:
+
+- Both the API and Admin projects must be running.
+- In the .env file of the API project, make sure the following environment variables are uncommented and set to matching values:
+  - `NOTIFY_E2E_TEST_EMAIL`
+  - `NOTIFY_E2E_TEST_PASSWORD`
+
+Example:
+
+```bash
+NOTIFY_E2E_TEST_EMAIL=your-email@example.com
+NOTIFY_E2E_TEST_PASSWORD=your-password
+```
+
+### How to Run BackstopJS
+
+#### Step 1: Install Gulp Globally
+
+First, install Gulp globally if its not installed:
+
+```
+npm install -g gulp
```

Comment:
Is this necessary to install globally, or would having `gulp` available in the project itself be enough?

Reply:
We just need to be able to run the command so having it in the project is enough if the command still works, ill test it.